lib/exercises.rb

```diff
@@ -23,8 +61,51 @@ def top_k_frequent_elements(list, k)
 #   Each element can either be a ".", or a digit 1-9
 #   The same digit cannot appear twice or more in the same 
 #   row, column or 3x3 subgrid
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: O(n), n is the length of table
+# Space Complexity: O(n), n is the length of table
 def valid_sudoku(table)
```

Comment:
Since the table is alway 9x9, the big-O is O(1).  

However I would also have taken O(n^2) where n is the length of a table since you hit every square.

Also avoid one letter variable names unless they are super clear.